src/main/java/com/minecolonies/sounds/CitizenSounds.java

```diff
@@ -0,0 +1,54 @@
+package com.minecolonies.sounds;
+
+import com.minecolonies.util.SoundUtils;
+import net.minecraft.util.SoundEvent;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import java.util.Random;
+
+/**
+ * Class used to store the basic citizen sounds.
+ */
+public class CitizenSounds
+{
+    /**
+     * Chance to say a phrase.
+     */
+    private static final int PHRASE_CHANCE = 50;
+
+    /**
+     * The citizen sound events for the females.
+     */
+    public static class Female
+    {
+        public static SoundEvent say;
+    }
+
+    /**
+     * The citizen sound events for the males.
+     */
+    public static class Male
+    {
+        public static SoundEvent say;
+    }
+
+    /**
+     * Plays fisherman sounds.
+     * @param worldIn the world to play the sound in.
+     * @param position the position to play the sound at.
+     * @param isFemale the gender.
+     */
+    public static void playCitizenSounds(World worldIn, BlockPos position, boolean isFemale)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Provide multiple methods instead of using "isFemale" to determine which action to take. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2301)